rebasehelper/helpers/lookaside_cache_helper.py

```diff
@@ -176,7 +177,7 @@ def post(check_only=False):
             sys.stdout.flush()
 
     @classmethod
-    def update_sources(cls, tool, basepath, package, old_sources, new_sources, upload=True):
+    def update_sources(cls, tool, basepath, package, old_sources, new_sources, upload=True, target=''):
```

Comment:
`target`, really? That doesn't sound right to me at all.

Reply:
You are right. What about `source_directory`? Or do you have something better in your mind?

Reply:
That's much better.